users/applicant_profiles/applicantProfileModel.js

```diff
@@ -19,6 +19,26 @@ function findApplicantProfileBy(filter) {
     .first();
 }
 
+//adds new applicant profile. This is only to be used during the onboarding process, function is inserted into add function in bothUserTypeModel file.
+async function addApplicantProfile(id) {
+
+  const defaultData = {
+    applicant_id: id,
```

Comment:
Lee,  I have a suggestion, instead of strings with space (city: " ",), I would use an empty string - city: "". It will be less surprising for people on the front end. What do you think?

Reply:
Good idea, I'll make that change!